src/visualizations/config/adapters/dhis_highcharts/legendSet.js

```diff
@@ -3,6 +3,8 @@ import i18n from '@dhis2/d2-i18n'
 
 import { getLegendByValueFromLegendSet } from '../../../../modules/legends'
 
+const OUT_OF_BOUNDS_COLOR = '#CCCCCC'
```

Comment:
Is this a specific color part of the "theme"?
Should it be imported from `@dhis2/ui`?

Reply:
Good point. Checked with @cooper-joe, the theme colors are meant for the interface, so we're sticking with a custom color in this case.